content/ru/multitenancy.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Мультитенантность
+status: Completed
+category: Property
+tags: ["architecture", "property", ""]
+---
+
+Мультитенантность (или мультиарендность) — это концепция, при которой одна инсталляция программного обеспечения обслуживает несколько арендаторов (тенантов).
+Арендатором может быть пользователь, приложение или группа пользователей/приложений, которые работают со своими собственными данными.
+Эти арендаторы не обмениваются данными (если только владелец не укажет иное) и могут даже не знать о существовании друг друга.
+
+Арендатор может быть как отдельным пользователем с одной учетной записью — например, в случае использования программного обеспечения 
+для решения персональных задач — так и целой корпорацией с тысячами учетных записей, у каждой из которых свои права доступа,
+но при этом они взаимосвязаны между собой множеством способов. 
+Такие программные средства как Google Mail, Google Docs, Microsoft Office 365, Salesforce CRM, Dropbox, а также многие другие,
+можно отнести к полностью или частично мультитенантному программному обеспечению.
+
+## Какую проблему решает
+
+Без мультитенантности каждому арендатору требовалась бы отдельная инсталляция программного обеспечения.
+Это увеличивало бы потребление ресурсов и затраты на обслуживание, а в конечном итоге и расходы на программное обеспечение.
+
+## Как именно решает проблему
+
+Мультитенантное программное обеспечение предоставляет каждому арендатору изолированную среду (рабочие данные, настройки, список учетных данных и т. д.),
+одновременно обслуживая нескольких арендаторов. 
+С точки зрения арендатора это выглядит как отдельная инсталляция программного обеспечения, хотя на самом деле все они используют одну и ту же.
+Для этого программное средство запускается на сервере, а арендаторам предоставляется возможность подключаться к нему через сеть, 
+используя интерфейс [API](/ru/application-programming-interface/) (смотри также [Архитектура клиент-сервер](/ru/client-server-architecture/)).
```

Comment:
An important detail is missing here. The original article states "via an interface and/or an API".
```suggestion
используя пользовательский интерфейс и/или программный ([API](/ru/application-programming-interface/); смотри также [Архитектура клиент-сервер](/ru/client-server-architecture/)).
```